.github/workflows/build-multiarch.yml

```diff
@@ -27,35 +27,40 @@ jobs:
           - java11-openj9
         include:
           - variant: java17
+            baseImage: eclipse-temurin:17-jre
+            platforms: linux/amd64,linux/arm/v7,linux/arm64
+            mcVersion: LATEST
+          - variant: java17jdk
             baseImage: eclipse-temurin:17
             platforms: linux/amd64,linux/arm/v7,linux/arm64
             mcVersion: LATEST
           - variant: java17-openj9
-            baseImage: ibm-semeru-runtimes:open-17-jdk
+            baseImage: ibm-semeru-runtimes:open-17-jre
             platforms: linux/amd64,linux/arm64
             mcVersion: LATEST
           - variant: java17-alpine
-            baseImage: eclipse-temurin:17-jdk-alpine
+            baseImage: eclipse-temurin:17-jre-alpine
             platforms: linux/amd64
             mcVersion: LATEST
           - variant: java8
```

Comment:
There also needed to be a `java8-jdk` variant added as a derivative of java8-multiarch.